backend/routes/sessionRoutes.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express'
+import handler from 'express-async-handler'
+const sessionRoutes = express.Router()
+import Session from '../models/sessionModel.js'
+
+//GET /sessions
+sessionRoutes.get('/', handler(async (req, res) => {
+    const sessions = await Session.find({})
+    res.json(sessions)
+}))
+
+//GET /sessions/:id
+sessionRoutes.get('/:id', handler(async (req, res) => {
+    const session = await Session.findById(req.params.id)
+    if (session) {
+        res.json(session)
+    } else {
+        res.status(404).json({ message: 'session not found' })
+    }
+}))
+
+//POST /sessions/create
+sessionRoutes.post('/create', handler(async (req, res) => {

```

Comment:
Not sure if my assumption is correct here, but if '/create' is part of a URI, I think we would need to change create to a noun because the general practice is to only have nouns for URIs.